boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -181,71 +181,31 @@ arduino_serial: &flexcomm12 {
 		regulator-initial-mode = <PCA9420_MODECFG0_PIN>;
 		modesel-reg = <PCA9420_MODECFG_0_0>;
 		modesel-mask = <PCA9420_MODECFG_0_MODE_CTRL_SEL_MASK>;
-		pca9420_sw1: sw1_buck {
-			voltage-range = <PCA9420_SW1_VOLTAGE_RANGE>;
-			current-levels = <PCA9420_CURRENT_LIMIT_LEVELS>;
-			num-voltages = <42>;
-			ilim-reg = <PCA9420_TOP_CNTL0>;
-			ilim-mask = <PCA9420_TOP_CNTL0_VIN_ILIM_SEL_MASK>;
-			num-current-levels = <7>;
-			vsel-reg = <PCA9420_MODECFG_0_0>;
-			vsel-mask = <PCA9420_MODECFG_0_SW1_OUT_MASK>;
-			enable-reg = <PCA9420_MODECFG_0_2>;
-			enable-mask = <PCA9420_MODECFG_2_SW1_EN_MASK>;
-			enable-val = <PCA9420_MODECFG_2_SW1_EN_VAL>;
-			min-uV = <500000>;
-			max-uV = <1800000>;
+		BUCK1 {
```

Comment:
A couple of comments on the node naming:
- why uppercase? This doesn't really fit with naming style for the rest of the devicetree
- I would like to preserve the node labels here, to make it easier for users to access these devices (unless we want to create a regulator specific macro to access voltage outputs via the PMIC IC node)

Reply:
> A couple of comments on the node naming:
> 
> * why uppercase? This doesn't really fit with naming style for the rest of the devicetree

So, the main reason here is Linux and our policy to be compatible with their bindings. If you check, most of their regulator bindings use uppercase for node names. The advantage of Linux is that they can enforce node names because their bindings' schema ([dt-schema](https://github.com/devicetree-org/dt-schema)) is much more powerful. I've done some initial work on adding support for such bindings language, but it's a big job and I have now other higher priorities... In fact, Zephyr normalizes all node names to lowercase, so in code we need to use e.g. `buck1` token and not `BUCK1`.

Examples:
https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/regulator/maxim,max14577.yaml#L52
https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/regulator/nxp,pca9450-regulator.yaml#L52

> * I would like to preserve the node labels here, to make it easier for users to access these devices (unless we want to create a regulator specific macro to access voltage outputs via the PMIC IC node)

I can leave nodelabels, sure. I just removed them since they were not used.

Reply:
> So, the main reason here is Linux and our policy to be compatible with their bindings. If you check, most of their regulator bindings use uppercase for node names.

Sounds good. Always happy to follow Linux bindings